etc/httpd/global.conf

```diff
@@ -875,10 +876,10 @@ RewriteRule ^/media/flash/playerWithControls.swf - [F]
 # Bug 896474
 RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?about(/?|/.+)$ /b/$1about$2 [PT]
 
-# Bug 987852
-RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?MPL(/?|/.+)$ /b/$1MPL$2 [PT]
-RewriteRule ^/MPL/1.1/index\.txt /media/MPL/1.1/index.txt [L,PT]
+# Bug 987852 & 1201914
+RewriteRule ^/(\w{2,3}(?:-\w{2})?/)?MPL/(.+).html$ /$1/MPL/$2/ [L,R=301]
```

Comment:
Oops. This is probably my fault. Shouldn't be a slash between `$1` and `MPL` in the destination: `/$1MPL/$2/`.